### DISCUSSION TITLE
[FormBuilderDropdown] How to hide an unknown value

### DISCUSSION THREAD

**Discussion by mareson** [danvick/flutter_form_builder#1242] [Q&A] [answered]
Hi,

I use dropdown so that the data it selects from is retrieved by a request from the server. For this reason I have implemented loading (which overlap FormBuilderDropdown). Unfortunately, the dropdown value is loaded before the items are loaded and it's loaded value (ID) is displayed on the screen in the dropdown (which is not very nice). I haven't found a way to not display this value, if appropriate item does not exists. Thank you very much for the answers.
<img width="399" alt="Snímek obrazovky 2023-05-10 v 14 46 24" src="https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/75789300/5db110cd-331b-426e-b575-39c6e9bf544e">
<img width="396" alt="Snímek obrazovky 2023-05-10 v 14 46 35" src="https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/75789300/7da6d95c-211e-4a0e-ba39-ce64ea9c6d27">

**Comment by deandreamatias** [comment, marked as answer]
This was a bug. I solve in this PR https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/pull/1255
After this change, you will can change to set a `disabledHint` or left the dropdown empty
